### ISSUE TITLE
Magnolia: could not find Schema.Typeclass for type zio.query.ZQuery

### ISSUE THREAD

**Issue by andrewpaprotsky** [ghostdogpr/caliban#580]
The following code no longer compiles:
```scala
package myapp

import caliban.schema.GenericSchema
import zio.blocking.Blocking
import zio.console.Console
import zio.query.ZQuery

case class Field(value: ZQuery[Console, Nothing, String])

case class Queries(field: ZQuery[Blocking, Nothing, Field])

object MySchema extends GenericSchema[Console with Blocking] {
  implicit lazy val queriesSchema = gen[Queries]
}
```
```
[error] /home/andrew/caliban/examples/src/main/scala/Issue.scala:13:40: magnolia: could not find Schema.Typeclass for type zio.query.ZQuery[zio.console.Console,Nothing,String]
[error]     in parameter 'value' of product type myapp.Field
[error]     in chained implicit Schema.Typeclass for type zio.query.ZQuery[zio.console.Console,Nothing,String]
[error]     in parameter 'value' of product type myapp.Field
[error]     in chained implicit Schema.Typeclass for type zio.query.ZQuery[zio.console.Console,Nothing,String]
[error]     in parameter 'value' of product type myapp.Field
[error]     in chained implicit Schema.Typeclass for type zio.query.ZQuery[zio.console.Console,Nothing,String]
[error]     in parameter 'value' of product type myapp.Field
[error]     in chained implicit MySchema.Typeclass for type zio.query.ZQuery[zio.blocking.Blocking,Nothing,myapp.Field]
[error]     in parameter 'field' of product type myapp.Queries
[error]   implicit lazy val queriesSchema = gen[Queries]
[error]                                        ^
[error] one error found
[error] (examples / Compile / compileIncremental) Compilation failed
```

It looks like this started to happen after #561. I wonder if `MySchema` should be defined in a different way to make things work.

**Comment by ghostdogpr** [comment]
Merged, it's available in `0.9.1+41-25032696-SNAPSHOT`.

**Comment by andrewpaprotsky** [comment]
Thanks! That was fast :)
